admin/installer/mysql/docker-compose-node1.yml

```diff
@@ -0,0 +1,10 @@
+database10:
```

Comment:
Thanks for pointing out! We used mysql as our endtoend test workload before moving on to postgres. Removed.